client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you-theme-section.tsx

```diff
@@ -143,6 +149,12 @@ export const ThankYouThemeSection = ( {
 		[ siteId, theme ]
 	);
 
+	useEffect( () => {
+		if ( isActive && continueWithPluginBundle ) {
+			page( `/setup/plugin-bundle/getCurrentThemeSoftwareSets?siteSlug=${ siteSlug }` );
```

Comment:
This automatically redirects the user after the theme is installed on the site (from Marketplace thank you page). Not the ideal UX, but I think it should be enough for now?

An alternative would be to add a new primary button after install, instead of the customize button. Something like "Continue with Store installation". However, if the user doesn't click on the button, the site will remain in a weird state where it's still on simple but with the Woo theme installed.

Reply:
Thanks for the clarifying comment. I agree this is not ideal UX as it seems confusing to the user. I would progress this task for now, but I would create a follow-up to discuss options. 

Some suggestions could be:
1. Add the "Continue with Store installation" button as you suggest
2. Display a message indicating that the redirect will happen after a few seconds, so the user has time to see what is going to happen

Additionally, from the main menu, the user can click on the `WooCommerce` menu item and `Start a new store` so if they close the page, they can always continue adding the store